### PR TITLE
Update django support

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -8,6 +8,8 @@ Unreleased
 * Dropped support for Python 3.5 and Django 1.11.
 * Fix crashing that could occur with ``similar_objects`` in multi-inheritance contexts.
 * Add Danish translation.
+* Remove Django 3.0 support.
```
Comment (auvipy) [inline CHANGELOG.rst:11]: support of django 3 can't be dropped unill django 2.2 retain. it is only removed from marix

**Comment by rtpg** [comment]
read through the 3.2 release notes and nothing stood out to me, so this looks good. Thanks for sending this in!

**Review by auvipy — APPROVED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (rtpg) [reply CHANGELOG.rst:11]: I thought Django 3.0 was out of support (whereas 2.2 is still LTS). I guess Django 3.0 will continue to work for a long time given how this stuff works and its proximity to 2.2, though. Am I missing some detail?

Comment (auvipy) [reply CHANGELOG.rst:11]: > I guess Django 3.0 will continue to work for a long time given how this stuff works and its proximity to 2.2, though

That is what I wanted to mean. no problem.

**Pull request merged**